vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/ScsiDiskControllerType.java

```diff
@@ -17,8 +17,10 @@
 package com.cloud.hypervisor.vmware.mo;
 
 public interface ScsiDiskControllerType {
-    public final static String LSILOGIC_PARALLEL = "lsilogic";
-    public final static String LSILOGIC_SAS = "lsisas1068";
-    public final static String BUSLOGIC = "buslogic";
-    public final static String VMWARE_PARAVIRTUAL = "pvscsi";
+    String LSILOGIC_PARALLEL = "lsilogic";
+    String LSILOGIC_SAS = "lsilogicsas";
+    String LSILOGIC_SAS_1068 = "lsisas1068";
```

Comment:
I think, only SCSI controllers are:
- BusLogic Parallel
- LSI Logic Parallel
- LSI Logic SAS
- Paravirtual

what's is the difference for 'lsilogicsas' and 'lsisas1068' here?

Reply:
@sureshanaparti that is correct but those strings are internally mapped by Vmware to the LSI Logic SAS controller. However, I could not find what the difference is between those values on the Vmware documentation

Reply:
@nvazquez Any strings there are internal to CloudStack? Check the SCSI controller types here:
- https://vmware.github.io/vsphere-automation-sdk-python/vsphere/cloud/com.vmware.vcenter.vm.hardware.html
- https://vmware.github.io/vsphere-automation-sdk-java/vsphere/6.5.0/vsphereautomation-client-sdk/com/vmware/vcenter/vm/hardware/adapter/ScsiTypes.Type.html

Reply:
@sureshanaparti I've reviewed the links, and I don't find this to be true in reality - please see this "table bellow" - without ACS, I've created a VM in vSpher 6.7, inspected the controller type in VMX, then exported to OVA, which sometimes had different controller type - please see. I also imported e.g. OVA with "lsilogicsas" and it became "lsisas1068" inside VMX:

vCenter UI                                                         VMX                                     OVF
 
BusLogic Parallel                              NA (not specified!)                         buslogic
LSI Logic Parallel                              lsilogic                                           lsilogic
LSI Logic SAS                                   lsisas1068                                     lsilogicsas
VMware Paravirtual                         pvscsi                                            VirtualSCSI

Reply:
ok @andrijapanicsb , thanks for the tests and confirmation.